### PR TITLE
Use checkout-code step

### PR DESCRIPTION
**Pull request by d2lam**
- Use the sd-checkout-code step
- ~~Expose doRun so we can add more logging inside launcher~~

Relates to https://github.com/screwdriver-cd/screwdriver/issues/312

Tested locally using in-a-box:
screen shot 2016-11-16 at 11 31 43 am

### CODE REVIEW

File: launch.go [hunk 1]
```diff
@@ -53,17 +52,13 @@ type scmPath struct {
...
 }
 
-func (s scmPath) httpsString() string {
-	return fmt.Sprintf("https://%s/%s/%s#%s", s.Host, s.Org, s.Repo, s.Branch)
-}
-
 // e.g. scmUri: "github:123456:master", scmName: "screwdriver-cd/launcher"
-func parseScmUri(scmUri, scmName string) (scmPath, error) {
-	uri := strings.Split(scmUri, ":")
+func parseScmURI(scmURI, scmName string) (scmPath, error) {
```
Comment (tkyi) [inline launch.go]: 👍 nice variable-naming convention catch

File: launch.go [hunk 2]
```diff
@@ -176,7 +171,7 @@ func launch(api screwdriver.API, buildID, rootDir, emitterPath string) error {
 		return fmt.Errorf("fetching Pipeline ID %q: %v", j.PipelineID, err)
 	}
 
-	scm, err := parseScmUri(p.ScmUri, p.ScmRepo.Name)
+	scm, err := parseScmURI(p.ScmUri, p.ScmRepo.Name)
```
Comment (tkyi) [inline launch.go]: should we change p.ScmUri to p.ScmURI to be consistent?
https://github.com/screwdriver-cd/launcher/blob/master/screwdriver/screwdriver.go#L94

Comment (d2lam) [inline launch.go]: Yeah let's do that.

File: launch.go [hunk 3]
```diff
@@ -187,36 +181,16 @@ func launch(api screwdriver.API, buildID, rootDir, emitterPath string) error {
 		return err
 	}
 
+	fmt.Fprintf(emitter, "Workspace created in %s\n", rootDir)
```
Comment (tkyi) [inline launch.go]: 👍 Much cleaner

**Pull request merged**